lib/liquid/standardfilters.rb

```diff
@@ -86,6 +86,7 @@ def truncatewords(input, words = 15, truncate_string = "...".freeze)
     #   <div class="summary">{{ post | split '//' | first }}</div>
     #
     def split(input, pattern)
+      pattern = pattern.to_s if pattern.is_a?(Numeric)
```

Comment:
What about for non-string values other than Numeric?  Should we raise a Liquid::Error for that so that it doesn't result in an internal error?  Maybe we should have a Liquid::Utils.to_string method to make that pattern easy.